tests/e2e/core-tests/specs/merchant/wp-admin-order-status-filters.test.js

```diff
@@ -4,178 +4,71 @@
  */
 const {
 	merchant,
-	createSimpleOrder,
+	batchCreateOrders,
 	clickFilter,
 	moveAllItemsToTrash,
 } = require( '@woocommerce/e2e-utils' );
 
 const statusColumnTextSelector = 'mark.order-status > span';
 
 // Define order statuses to filter against
-const orderStatus = {
-	pending: {
-		name: 'wc-pending',
-		description: { text: 'Pending payment' },
-	},
-	processing: {
-		name: 'wc-processing',
-		description: { text: 'Processing' },
-	},
-	onHold: {
-		name: 'wc-on-hold',
-		description: { text: 'On hold' },
-	},
-	completed: {
-		name: 'wc-completed',
-		description: { text: 'Completed' },
-	},
-	cancelled: {
-		name: 'wc-cancelled',
-		description: { text: 'Cancelled' },
-	},
-	refunded: {
-		name: 'wc-refunded',
-		description: { text: 'Refunded' },
-	},
-	failed: {
-		name: 'wc-failed',
-		description: { text: 'Failed' },
-	}
-};
+const orderStatus = [
+	['Pending payment', 'wc-pending'],
+	['Processing', 'wc-processing'],
+	['On hold', 'wc-on-hold'],
+	['Completed', 'wc-completed'],
+	['Cancelled', 'wc-cancelled'],
+	['Refunded', 'wc-refunded'],
+	['Failed', 'wc-failed']
+];
```

Comment:
I had to refactor this variable so it's easier to access its values from the `each`'ed test.